gamla/functional_test.py

```diff
@@ -760,3 +760,10 @@ def test_sample_with_randint():
 
 def test_duplicated_values():
     assert functional.duplicated_values([1, 2, "a", "a", 5]) == ("a",)
+
+
+def test_ignore_first():
```

Comment:
you have a test like this in higher_order_test. Maybe you meant to test make_key ?